src/travis2docker/travis2docker.py

```diff
@@ -119,8 +119,7 @@ def _compute(self, section):
         job_method = getattr(self, '_compute_' + section_type)
         return job_method(section_data, section)
 
-    @staticmethod
-    def _compute_env(data, _):
+    def _compute_env(self, data, _):
```

Comment:
Is there a way to use static method yet here?

Reply:
@moylop260 
Sorry, but I think this is the only way. I considered:
- Adding a new method: increases the complexity, because it has to run before environment is computed, so some validations would need to be done twice.
- Using the `compute_addon`: wouln't work, because `compute_env` runs before anything else.

Take into account this is the only compute method that is not static.